lib/index.js

```diff
@@ -79,9 +59,12 @@ const init = async function (options = {}) {
   }
 
   const server = await Steerage.init({
+    basedir: baseDir,
     config: Path.join(__dirname, 'config.json'),
+    environment,
     onconfig: onConfig,
-    environment
+    protocols: shortstopHandlers,
+    userConfigPaths: [...(userConfigPath ? [].concat(userConfigPath) : [])]
```

Comment:
convert to an array regardless of whether a string or an array is provided to userConfigPath for backwards compatibility.